chat_shell/chat_shell/services/streaming/core.py

```
@@ -27,6 +27,17 @@
 REASONING_START = "__REASONING__"
 REASONING_END = "__END_REASONING__"
 
+# Constants for truncation markers (matching graph_builder.py)
+TRUNCATED_START = "__TRUNCATED__"
+TRUNCATED_END = "__END_TRUNCATED__"
+
+# Bilingual truncation warning message
+TRUNCATION_WARNING_MESSAGE = """
+
+---
+⚠️ 内容已截断：模型输出达到 token 上限，响应可能不完整。
```

Ruff (0.14.14)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)